app_server/app/adapters/socket/Room.js

```diff
@@ -75,4 +75,21 @@ Class.removeAllClients = function() {
   this.__clients = {};
 };
 
+Class.getNumberOfUsers = function() {
+  let users = {};
+  let count = 0;
+  for (var socketId in this.getClients()) {
+    let userId = this.getClient(socketId).getUserId(); 
+    if (!users[userId]) {
+      users[userId] = true;
+      count++;
+    }
+  }
+  return count;
```

Comment:
Hmm not very safe to do `filter` here since the `users` map is updated dynamically during filtering so I will leave this like this for now.